resources/aws/consts.go

```diff
@@ -5,9 +5,12 @@ const (
 	tagKeyName    string = "Name"
 	tagKeyCluster string = "Cluster"
 	// Subnet keys
-	subnetAvailabilityZone string = "availabilityZone"
-	subnetCidrBlock        string = "cidrBlock"
-	subnetDescription      string = "description"
-	subnetGroupName        string = "group-name"
-	subnetVpcID            string = "vpc-id"
+	subnetAvailabilityZone     string = "availabilityZone"
+	subnetCidrBlock            string = "cidrBlock"
+	subnetDescription          string = "description"
+	subnetGroupName            string = "group-name"
+	subnetVpcID                string = "vpc-id"
+	subnetIPPermissionCIDR     string = "ip-permission.cidr"
+	subnetIPPermissionFromPort string = "ip-permission.from-port"
+	subnetIPPermissionToPort   string = "ip-permission.to-port"
```

Comment:
Is it common to put all kinds of constants into one separate file? I thought they should go to where they are actually used. Uncertain.